src/usb/sagas.ts

```diff
@@ -164,10 +164,26 @@ function* handleUsbConnectPybricks(hotPlugDevice?: USBDevice): Generator {
     // REVISIT: For now, we are making the assumption that there is only one
     // configuration and it is already selected and that it contains a Pybricks
     // interface.
-    assert(
-        usbDevice.configuration !== undefined,
-        'USB device configuration is undefined',
-    );
+
+    if (!usbDevice.configuration && usbDevice.configurations.length > 0) {
+        const [, selectErr] = yield* call(() =>
+            maybe(
+                usbDevice.selectConfiguration(
+                    usbDevice.configurations[0].configurationValue,
+                ),
+            ),
+        );
```

Comment:
```suggestion
    if (usbDevice.configuration === null) {
        const [, selectErr] = yield* call(() =>
            maybe(usbDevice.selectConfiguration(1)),
        );
```

Since we control the firmware, we know that we always want configuration 1, which may or may not be the configuration at index 0.

And prefer strict equality checks instead of truthy/falsy unless there is a good reason for it.

Reply:
ok, corrected.

Any comment or assert to document this "magic" number?

Concern: code-logic wise this could even break, only works as you control the firmware.